### PR TITLE
PEPPER-1365 . Special handling of lms release v2 pdfs config with consent v1

### PR DESCRIPTION
**Pull request by ssettipalli**
PEPPER-1365
Handle scenarios where lms user completed release v2 but had old consent v1.

Tried the approach to update release v2 pdf config to remove reference to consent v1 . Approach was clean with one issue for DSM. Since MEDICAL_RELEASE activity was same for all self/parental and consent-assent, DataExporter was updating elastic with ALL 3 release pdfs and DSM is displaying ALL 3 pdf config names . 
Basically DSM displays some additional invalid release configs as download options. 
To avoid that issue and confusion ended with this hardcoded special handling similar to what we did for pancan. 

Other option is add special handling to remove other pdf config names which is equally hardcoded/ugly.
With this way, no pdfconfig changes (studybuilder patch) needed atleast and less invasive

### CODE REVIEW

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java [hunk 1]
```diff
@@ -971,6 +977,13 @@ private List<PdfConfigInfo> findPdfConfigsForStudyUser(List<PdfConfigInfo> study
             }
         }
 
+        //lms special case to handle scenarios where lms user consented-v1 but had release-v2
+        if (!studyConfigs.isEmpty() && studyConfigs.get(0).getStudyGuid().equals(LMS_GUID)) {
```
Comment (ssettipalli) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java:981]: special handling of pdf config with users consented - v1 and completed newer release - v2

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java [hunk 2]
```diff
@@ -994,6 +1007,25 @@ private PdfConfigInfo getPancanReleaseV2PdfConfigIfNeeded(
         return releasePdfConfig;
     }
 
+    private PdfConfigInfo getLmsReleaseV2PdfConfigIfNeeded(
```
Comment (ssettipalli) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/DataExporter.java:1010]: looked into other pancan special handling code added recently to avoid code duplication but activity names, pdf config names being different didn't make it feasible

**Review by zyme — APPROVED** [review]
LGTM
